oss-internship-2020/sapi_lodepng/examples/main_sandboxed.cc

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <glog/logging.h>
+
+#include <iostream>
+
+#include "helpers.h"
+#include "sandbox.h"
+
+void EncodeDecodeOneStep(SapiLodepngSandbox &sandbox, LodepngApi &api) {
```

Comment:
The C++ style guide allows both, but Sandboxed API prefers to keep `&` and `*` close to the type.
```c++
void EncodeDecodeOneStep(SapiLodepngSandbox& sandbox, LodepngApi& api) {
```

Change here and everywhere else in your PR.

You can also add these two lines to `.clang-format` to have clang-format account for this (change in SAPI is on the way):
```
DerivePointerAlignment: false
PointerAlignment: left
```